librpc/Rpc.cpp

```diff
@@ -615,6 +618,124 @@ Json::Value Rpc::getBlockByHash(
 }
 
 
+void Rpc::generateBlockHeaderInfo(Json::Value& _response,
+    std::shared_ptr<
+        std::pair<std::shared_ptr<dev::eth::BlockHeader>, dev::eth::Block::SigListPtrType>>
+        _headerInfo,
+    bool _includeSigList)
+{
+    auto blockHeader = _headerInfo->first;
```

Comment:
Codacy found an issue: [Either the condition '!blockHeaderInfo' is redundant or there is possible null pointer dereference: _headerInfo.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5987732)